plugins/remsql/fdb_comm.c

```diff
@@ -1014,6 +1016,9 @@ int fdb_msg_read_message(SBUF2 *sb, fdb_msg_t *msg, enum recv_flags flags)
         } else {
             msg->co.srcname = NULL;
         }
+
+        if (!fdb_is_dbname_in_whitelist(msg->co.srcname))
+            return -1;
```

Comment:
this has to pass an error and that gets converted to SQLITE_ACCESS

Reply:
Done.